react/features/remote-video-menu/components/web/MuteEveryoneElseButton.js

```diff
@@ -0,0 +1,77 @@
+/* @flow */
+
+import React from 'react';
+
+import { createToolbarEvent, sendAnalytics } from '../../../analytics';
+import { openDialog } from '../../../base/dialog';
+import { translate } from '../../../base/i18n';
+import { IconMicDisabled } from '../../../base/icons';
+import { connect } from '../../../base/redux';
+
+import AbstractMuteButton, {
+    _mapStateToProps,
+    type Props
+} from '../AbstractMuteButton';
+
+import RemoteVideoMenuButton from './RemoteVideoMenuButton';
+import MuteEveryoneDialog from '../../../remote-video-menu/components/web/MuteEveryoneDialog';
+
+/**
```

Comment:
This JSDoc is copy/paste from somewhere else and it doesn't explain anything about the component.

Reply:
updated